### PR TITLE
TE-1647 Hide gallery with less than two images

### PR DESCRIPTION
**Pull request by red-runner**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-1647)

### What **one** thing does this PR do?
Hides the `PropertyPageHero` gallery when there are fewer than two images

### Behaviour
screenshot 2019-01-21 at 12 32 50

### CODE REVIEW

File: src/components/property-page-widgets/PropertyPageHero/component.js [hunk 1]
```diff
@@ -103,7 +83,7 @@ Component.defaultProps = {
   propertyName: null,
   ratingNumber: null,
   searchBarGetIsDayBlocked: undefined,
-  searchBarModalHeadingText: undefined,
+  searchBarModalHeadingText: '',
```
Comment (red-runner) [inline src/components/property-page-widgets/PropertyPageHero/component.js]: @marcdavi-es @Jonnygwi I was getting a warning about the `Heading` component not having any children. This solved the warning, although I'm not sure if it's the right solution. Have a look.

Comment (marcdavi-es) [reply src/components/property-page-widgets/PropertyPageHero/component.js]: In `HomepageHero` we have

```js
searchBarModalHeadingText: CHECK_OUR_AVAILABILITY,
```

I think that works OK here too?

Comment (red-runner) [reply src/components/property-page-widgets/PropertyPageHero/component.js]: Yeah, this will work better 🎉

**Pull request merged**